modules/odc-deploying-java-applications.adoc

```diff
@@ -26,7 +26,7 @@ Use the following instructions to upload a JAR file in the *Topology* view to de
 
 . In the *Name* field, enter a unique component name to name the associated resources.
 
-. In the *Resources* field, choose the resource type for your application.
+. Optional: For the *Advanced options* -> *Resource type*, you can set the default resource type from the *User Preferences* page. Also, you can modify the default selected resource type by using the *Resource type* drop-down list. 
```

Comment:
```suggestion
. Optional: You can select a different resource type from the default resource type if needed from the Advanced options -> Resource type drop-down list.
```

Reply:
@smunje1 selecting a default resource type from user preferences page should already be there in the user-preferences doc. Can you check and confirm? If not then we can include that here.

Reply:
@divyanshiGupta Yes, you are right. It seems we have already covered selecting a default resource type from the user preferences page as part of RHDEVDOCS-4236 https://github.com/openshift/openshift-docs/pull/49691